packages/browser-wallet/src/popup/pages/Account/AccountSettings/ConnectedSites.tsx

```diff
@@ -1,46 +1,99 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useAtom, useAtomValue } from 'jotai';
 import { selectedAccountAtom, storedConnectedSitesAtom } from '@popup/store/account';
 import Button from '@popup/shared/Button';
 import { useTranslation } from 'react-i18next';
+import { popupMessageHandler } from '@popup/shared/message-handler';
+import { EventType } from '@concordium/browser-wallet-api-helpers';
 
 export default function ConnectedSites() {
     const { t } = useTranslation('account', { keyPrefix: 'settings.connectedSites' });
     const selectedAccount = useAtomValue(selectedAccountAtom);
     const [connectedSitesLoading, setConnectedSites] = useAtom(storedConnectedSitesAtom);
     const connectedSites = connectedSitesLoading.value;
+    const [openTabUrl, setOpenTabUrl] = useState<string>();
+
+    async function getCurrentOpenTabUrl() {
+        const tabs = await chrome.tabs.query({ active: true, lastFocusedWindow: true });
+        const { url } = tabs[0];
+        if (!url) {
+            return undefined;
+        }
+        return new URL(url).origin;
```

Comment:
The origin differentiates between http and https, but maybe that is preferable?

Reply:
We will keep it as-is as that is more restrictive.